libraries/core-react/src/components/Button/Button.stories.tsx

```diff
@@ -314,18 +322,27 @@ export const FullWidth: Story<ButtonProps> = () => (
   </FullWidthWrapper>
 )
 
-export const Compact: Story<ButtonProps> = () => (
-  <EdsProvider density="compact">
-    <Wrapper>
-      <Button>Contained</Button>
-      <Button variant="outlined">Outlined</Button>
-      <Button variant="ghost">Ghost</Button>
-      <Button variant="ghost_icon">
-        <Icon data={menu} title="Ghost icon menu"></Icon>
-      </Button>
-    </Wrapper>
-  </EdsProvider>
-)
+export const Compact: Story<ButtonProps> = () => {
+  const [density, setDensity] = useState<Density>('comfortable')
+
+  useEffect(() => {
+    // Simulate user change
+    setDensity('compact')
+  }, [density])
+
+  return (
+    <EdsProvider density={density}>
+      <Wrapper>
+        <Button>Contained</Button>
+        <Button variant="outlined">Outlined</Button>
+        <Button variant="ghost">Ghost</Button>
+        <Button variant="ghost_icon">
+          <Icon data={menu} title="Ghost icon menu"></Icon>
+        </Button>
+      </Wrapper>
+    </EdsProvider>
+  )
+}
```

Comment:
This should not be a changed in this PR or was this intentional?

Reply:
This was related to the conflict. I performed rebased, now everything is in check.

Reply:
@laliq Are you sure you pulled the latest changes from develop locally before rebasing it onto this branch? We are missing from develop the past 5 days 🤔

Reply:
running merge now

Reply:
@mimarz looks good to me now

Reply:
Now you have 73 files flagged as changed... 🙈

Reply:
I did this...
```
git checkout my-brunch
git merge origin/master
```
I guess it was wrong 😑

Reply:
```sh
git rebase develop
git push -f
```